src/server.ts

```diff
@@ -1,13 +1,17 @@
-import * as express from 'express';
-import * as cors from 'cors';
-import routers from './routes';
-
-const app = express();
-const PORT = process.env.APP_PORT || 5000;
-
-app.use(express.json());
-app.use(cors());
-app.use(routers);
-app.listen(PORT, () => {
-    console.log(`Listening on port ${PORT}`)
-});
+import * as express from 'express';
+import * as cors from 'cors';
+import * as morgan from 'morgan';
+import routers from './routes';
+import handleErrors from './utils/errors';
+
+const app = express();
+const PORT = process.env.APP_PORT || 5000;
+
+app.use(morgan('combined'));
+app.use(express.json());
+app.use(cors());
+app.use(handleErrors);

```

Comment:
Boa galera. Só uma dúvida. O Saleh comentou que o handler de errors tem que estar declarado abaixo das rotas no server para funcionar. Vejam isso pra mim, e arrumem se for o caso pfvr

Reply:
E tem conflito aqui também. Dêem uma olhada por favor

Reply:
Done